src/stan/io/stan_csv_reader.hpp

```diff
@@ -2,7 +2,7 @@
 #define STAN_IO_STAN_CSV_READER_HPP
 
 #include <boost/algorithm/string.hpp>
-#include <stan/math/prim/mat/fun/Eigen.hpp>
+#include <stan/math/prim.hpp>
```

Comment:
See previous comment. Since there are no other functions used from prim, perhaps including only the Eigen header is better? I have no strong feelings either way.